graders/python/python_autograder/pl_main.py

```diff
@@ -91,6 +93,7 @@ def add_files(results):
         # Compile total number of points
         max_points = test_case.get_total_points()
         earned_points = sum([test["points"] for test in results])
+        score = 0 if max_points == 0 else float(earned_points) / float(max_points)
```

Comment:
Nit:
```suggestion
        score = 0 if float(max_points) == 0 else float(earned_points) / float(max_points)
```

Reply:
If you do this, make sure to use `math.isclose` instead of `==`.

Reply:
Oh? Say more, I'm curious when a float value would not equal zero but also cause a `ZeroDivisionError`.

Reply:
It won't throw the exception, but a good general rule to always avoid comparing floats using `==`. It probably doesn't matter in this case, though it might if `max_points` is a computed value.

Reply:
IMO we don't care if `max_points` is _almost_ zero, which could (in some very strange situation) be valid. Only if it's actually zero, in which case we need to avoid a divide-by-zero situation. Does that sound right?

Reply:
That makes sense 👍